tests/platform_tests/api/test_fan.py

```diff
@@ -115,6 +165,8 @@ def test_get_speed(self, duthost, localhost, platform_api_conn):
                 if self.expect(isinstance(speed, int), "Fan {} speed appears incorrect".format(i)):
                     self.expect(speed > 0 and speed <= 100,
                                 "Fan {} speed {} reading is not within range".format(i, speed))
+                    self.compare_value_with_platform_facts('speed', speed)
```

Comment:
`speed` will vary by device, so we are not expecting this value to be stored in the platform.json file. We should not perform a comparison here.

Reply:
fixed